src/controllers/organizations.js

```diff
@@ -0,0 +1,176 @@
+const { default: mongoose } = require('mongoose');
+const { Organization, User } = require('../models/user');
+const Fund = require('../models/fund');
+
+async function getOneOrganization(req) {
+    const { id: orgaId } = req.params;
+    const requiredUserFields = [
+        'id',
+        'firstName',
+        'lastName',
+        'email',
+        'profileImage',
+    ];
+    const requiredFundFields = [
+        'id',
+        'title',
+        'content',
+        'categories',
+        'targetFund',
+        'gatheredFund',
+        'address',
+        'createdAt',
+        'remainingFund',
+    ];
+    const requiredEventFields = [
+        'id',
+        'title',
+        'content',
+        'coverImage',
+        'date',
+        'categories',
+        'confirmedVolunteers',
+        'invitedVolunteers',
+        'address',
+        'location',
+        'createdAt',
+    ];
+
+    const requiredOrgaFields = {
```

Comment:
for me i used a short cut using only excluded fieldes! hahah 
`const excludeFields = { hashedPassword: 0, provider: 0, isVerified: 0 };`
and add it to second argument in your mongoose function.

Reply:
@sarahmood, you are right, but I will add also the email and the providerId to the excluded fields since they are private data to the organization (after rating the sent data is visible to anyone on the site.)